cmd/kube-scheduler/app/server.go

```diff
@@ -123,7 +95,38 @@ through the API as necessary.`,
 }
 
 // Run runs the Scheduler.
```

Comment:
Please change code comment to lowercase also.

Reply:
Done! 👍